relay-event-normalization/src/event.rs

```diff
@@ -1109,6 +1109,80 @@ mod tests {
         RawUserAgentInfo,
     };
 
+    const IOS_MOBILE_EVENT: &str = r#"
+        {
+            "sdk": {"name": "sentry.cocoa"},
+            "contexts": {
+                "trace": {
+                    "op": "ui.load"
+                }
+            },
+            "measurements": {
+                "app_start_warm": {
+                    "value": 8049.345970153808,
+                    "unit": "millisecond"
+                },
+                "time_to_full_display": {
+                    "value": 8240.571022033691,
+                    "unit": "millisecond"
+                },
+                "time_to_initial_display": {
+                    "value": 8049.345970153808,
+                    "unit": "millisecond"
+                }
+            }
+        }
+        "#;
+
+    const ANDROID_MOBILE_EVENT: &str = r#"
```

Comment:
The iOS event has 3 measurements. Does the minimal Android event need 9 measurements, or can we reduce it?

Reply:
Good point, I removed a number of the measurements. I had them in because they were measurements from a real Sentry event, but they aren't necessary for the minimal test input.